src/coreclr/src/jit/compiler.cpp

```diff
@@ -4269,16 +4274,213 @@ void Compiler::compCompile(void** methodCodePtr, ULONG* methodCodeSize, JitFlags
 
     EndPhase(PHASE_POST_IMPORT);
 
-    /* Initialize the BlockSet epoch */
-
+    // Initialize the BlockSet epoch
     NewBasicBlockEpoch();
 
-    /* Massage the trees so that we can generate code out of them */
+    // Start of the code that used to make up fgMorph
```

Comment:
I'd avoid mentioning what used to be, and instead encompass the comment below, to say something like: "Start of the code that is broadly called morphing, and includes global morph, as well as other phases that massage the trees so that we can generate code out of them."

Reply:
Updated